src/python_qt_binding/binding_helper.py

```diff
@@ -235,7 +238,7 @@ def createWidget(self, class_name, parent=None, name=''):
         # but this does not work in PySide 1.0.6: it simply segfaults...
         #loader = CustomUiLoader(baseinstance)
         #custom_widgets = custom_widgets or {}
-        #for custom_widget in custom_widgets.values():
+        # for custom_widget in custom_widgets.values():
```

Comment:
I was wondering why only this line was affected but not the two lines above?

Reply:
I'm really not sure about the internals of autopep8. It seems to consistently do about 95% of what you would hope it would do